### DISCUSSION TITLE
Option for users to change eggs at any time without having to be an administrator

### DISCUSSION THREAD

**Discussion by MrMoreira** [pterodactyl/panel#4833] [Feature Requests]
I would like to thank all of you for the incredible work of developing the pterodactyl panel, I know it is difficult because I am starting in the field of programming now, and I would like to ask you please, if you could implement an option in the control panel that allows any user swap pterodactyl eggs at any time.

**Comment by crackscout123** [comment]
Though about this but i think this would be a security concern cause the eggs are public to all users so if one of the users changes stuff it would apply to all instances when some hits the reinstall button.

**Reply by danny6167** [reply]
That's not what they are asking for. They want the ability for users to change their server to different eggs.

**Comment by danny6167** [comment]
It's been requested a couple of time, but I don't believe there is any intention to implement this into Pterodactyl.
There are a few egg switcher mods on the market you might want to look for if you need something like this.

**Comment by crackscout123** [comment]
What about the pelican.dev panel? I’m pretty sure there you can easily implement it due to plugin support.

Am 13.06.2024 um 03:57 schrieb Daniel Barton ***@***.***>:

﻿

That's not what they are asking for. They want the ability for users to change their server to different eggs.

**Reply by danny6167** [reply]
You would have to ask Pelican about their software. We have nothing to do with them.
But last I knew, their plugin system wasn't even half built yet.
